### PR TITLE
Added three new papers

### PR DESCRIPTION
**Pull request by sairajk**

### CODE REVIEW

**Comment by 53X** [comment]
@sairajk , please write the names of the papers that you added as the description to this PR

File: README.md [hunk 1]
```diff
@@ -86,6 +86,9 @@ This is a work in progress, so if you know of Twitter tools that are not mention
 + [Personalized Recommendation of Twitter Lists Using Content and Network Information](https://www.aaai.org/ocs/index.php/ICWSM/ICWSM14/paper/download/8120/8142)
 + [Detecting Automation of Twitter Accounts: Are You a Human, Bot, or Cyborg?](https://www.ijirset.com/upload/2017/august/286_Implemnattaion_IEEE.pdf)
 + [Fame for sale: Efficient detection of fake Twitter followers](http://daneshyari.com/article/preview/553058.pdf)
++ [Sentiment Analysis of Twitter Data: A Survey of Techniques](https://arxiv.org/abs/1601.06971)
++ [Techniques for sentiment analysis of Twitter data: A comprehensive survey](https://ieeexplore.ieee.org/document/7813707/)
++ [Like It or Not: A Survey of Twitter Sentiment Analysis Methods](https://www.researchgate.net/publication/304916478_Like_It_or_Not_A_Survey_of_Twitter_Sentiment_Analysis_Methods)
```
Comment (53X) [inline README.md]: Nice add =)

File: README.md [hunk 2]
```diff
@@ -86,6 +86,9 @@ This is a work in progress, so if you know of Twitter tools that are not mention
 + [Personalized Recommendation of Twitter Lists Using Content and Network Information](https://www.aaai.org/ocs/index.php/ICWSM/ICWSM14/paper/download/8120/8142)
 + [Detecting Automation of Twitter Accounts: Are You a Human, Bot, or Cyborg?](https://www.ijirset.com/upload/2017/august/286_Implemnattaion_IEEE.pdf)
 + [Fame for sale: Efficient detection of fake Twitter followers](http://daneshyari.com/article/preview/553058.pdf)
++ [Sentiment Analysis of Twitter Data: A Survey of Techniques](https://arxiv.org/abs/1601.06971)
```
Comment (53X) [inline README.md]: Please remove this one. There is no point in adding multiple survey papers on the same topic

File: README.md [hunk 3]
```diff
@@ -86,6 +86,9 @@ This is a work in progress, so if you know of Twitter tools that are not mention
...
++ [Techniques for sentiment analysis of Twitter data: A comprehensive survey](https://ieeexplore.ieee.org/document/7813707/)
```
Comment (53X) [inline README.md]: This is not a good paper. Remove this too.

**Comment by sairajk** [comment]
Removed the 2 papers that you mentioned. And added one more.

**Comment by 53X** [comment]
Thanks for the contribution

**Pull request merged**